apps/web/components/booking/CancelBooking.tsx

```diff
@@ -89,6 +103,7 @@ export default function CancelBooking(props: Props) {
               </Button>
               <Button
                 data-testid="confirm_cancel"
+                disabled={isBookingCancellationReasonRequired && !cancellationReason}
```

Comment:
Does this work? `cancellationReason` is only a string type, so how does it work with `!cancellationReason`? Instead, we should check the length.

Reply:
Empty string ("") is considered as undefined

![Screenshot 2024-09-23 105351](https://github.com/user-attachments/assets/b6bf627f-144f-432e-870d-00843b4939cd)